src/components/dashboard/GapByRace.vue

```diff
@@ -1,26 +1,110 @@
 <template>
-  <div class="gap-container">
-    <div>
-      <GapChart
-        :active-stats="activeStats"
-        :expected="expected"
-        :field-data="fieldData"
-      />
+  <!--
+    Depending on the controls, display different charts and layouts
+
+    Implementation:
+    Each child of the outer-container is stacked on top of each other taking up space and his hidden based on the
```

Comment:
```suggestion
    Each child of the outer-container is stacked on top of each other taking up space and is hidden based on the
```

new lines could maybe use reformatting as well, looks a bit overflowy